cli/cmd/proxy/loglevel/command.go

```diff
@@ -0,0 +1,318 @@
+package loglevel
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+	"sync"
+
+	"github.com/hashicorp/consul-k8s/cli/common"
+	"github.com/hashicorp/consul-k8s/cli/common/flag"
+	"github.com/hashicorp/consul-k8s/cli/common/terminal"
+	"github.com/posener/complete"
+	helmCLI "helm.sh/helm/v3/pkg/cli"
+	"k8s.io/apimachinery/pkg/api/validation"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+const (
+	defaultAdminPort    = 19000
+	flagNameNamespace   = "namespace"
+	flagNameKubeConfig  = "kubeconfig"
+	flagNameKubeContext = "context"
+)
+
+type LoggerConfig map[string]string
+
+var ErrMissingPodName = errors.New("Exactly one positional argument is required: <pod-name>")
+
+var levelToColor = map[string]string{
```

Comment:
colors here are based on the color scheme from https://github.com/hashicorp/go-hclog/blob/fb16e2d8d777021c96a0eb76290e59bc1c810bcd/intlogger.go#L47-L51